src/Microsoft.Fast.Components.FluentUI/DesignToken.cs

```diff
@@ -0,0 +1,110 @@
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+
+namespace Microsoft.Fast.Components.FluentUI
+{
+    public class DesignToken<T> : IAsyncDisposable
+    {
+        private readonly Lazy<Task<IJSObjectReference>> moduleTask;
+       
+        [Parameter]
+        public T? Value { get; set; }
+
+        [Parameter]
+        public T? DefaultValue { get; set; }
+
+        [Parameter]
+        public string? Name { get; set; }
+
+        /// <summary>
+        /// Constructs an instance of a DesignToken"/>.
+        /// </summary>
+        public DesignToken(IJSRuntime jsRuntime)
+        {
+            moduleTask = new(() => jsRuntime.InvokeAsync<IJSObjectReference>(
```

Comment:
Is it ok to do this for every token instance?

Reply:
I think so, but maybe ask ASP team for review / guidance

Reply:
@javiercn Would you mind taking a look at this and giving us your thoughts?

Reply:
@javiercn how do you feel about this?  Thx

Reply:
Yes, it is ok to do this for every token as they are properties of the `DesignTokens` class that need to be initialized.